helm/ffc-mpdp-backend/values.yaml

```diff
@@ -43,3 +43,16 @@ readinessProbe:
   initialDelaySeconds: 10
   periodSeconds: 10
   failureThreshold: 3
+
+postgresService:
+  postgresDb: ffc_mpdp_backend
+  postgresqlHost: ffc-mpdp-backend
+  postgresPort: 5432
+  postgresSchema: public
+  postgresUser: postgres
```

Comment:
These values will get overwritten from equivalent values set in Azure App Configuration by pipeline magic.  Need to set those values up ahead of time.